src/components/journeys/JourneysInstancesDataTable.spec.tsx

```diff
@@ -0,0 +1,93 @@
+import Chance from 'chance';
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+dayjs.extend(relativeTime);
+
+import mockJourneyInstance from 'utils/testing/mocks/mockJourneyInstance';
+import mockPerson from 'utils/testing/mocks/mockPerson';
+import { ZetkinJourneyInstance } from 'types/zetkin';
+
+const chance = Chance();
+
+const ids = Array.from(Array(100).keys());
+const people = ids
+  .slice(1, 20)
+  .map((id) =>
+    mockPerson({ first_name: chance.first(), id, last_name: chance.last() })
+  );
+const milestones = [
+  'make coffee',
+  'prepare for battle',
+  'perform lip sync',
+  'sashay away',
+];
+
+const valueTags = [
+  {
+    tags: [
+      { color: 'salmon', value: '1 - immediate' },
+      { color: 'peach', value: '2 - near future' },
+      { color: 'grey', value: '3 - chase up' },
+    ],
+    title: 'priority',
+  },
+  {
+    tags: [
+      { color: 'beige', value: 'contract' },
+      { color: 'cornflowerblue', value: 'pay' },
+      { color: 'grey', value: 'disciplinary/dismissal' },
+      { color: 'aliceblue', value: 'discrimination' },
+      { color: 'aquamarine', value: 'whistleblowing' },
+    ],
+    title: 'category',
+  },
+];
```

Comment:
Cool use of randomizing in this file! I'm not familiar with `chance` so I don't know how data needs to be formatted to be used with it. But on the Zetkin side of things, a value tag will always have the same color, regardless of it's value.

It looks to me like the tags in this dummy data are inspired by the examples in the design. If that's the case, I also want to point out that neither "priority" nor "category" would be (or are depicted in Figma) as value tags. They're tag groups.

If you want, we can have a chat today to make sure we have the same understanding of the data model.